source/fundamentals/builders/sort.txt

```diff
@@ -11,3 +11,347 @@ Sorts Builders
+have fields with identical values you can specify additional fields to sort
+on in the event of a tie. 
+
+The following code snippet shows how to specify a descending
+sort on the ``letter`` field followed by the ``_id`` field: 
+
+.. code-block:: java
+
+   import static com.mongodb.client.model.Sorts.descending;
+
+   // <MongoCollection setup code here>
+   
+   collection.find().sort(descending("letter", "_id"));
+
+
+The code snippet above returns the documents in the
+:ref:`sort_example collection <sorts-builders-sort-example>`  
+in the following order: 
+
+.. code-block:: json
+
+   {"_id": 6, "letter": "c", "food": "maple donut"}
+   {"_id": 1, "letter": "c", "food": "coffee with milk"}
+   {"_id": 4, "letter": "b", "food": "coffee with sugar"}
+   {"_id": 5, "letter": "a", "food": "milk and cookies"}
+   {"_id": 3, "letter": "a", "food": "maple syrup"}
+   {"_id": 2, "letter": "a", "food": "donuts and coffee"}
+
+Combining Sort Criteria
+-----------------------
+
+To combine sort criteria, use the ``Sorts.orderBy()`` static factory
+method. The ``orderBy()`` method applies additional sort criteria from
+left to right in the event of ties. 
+
+In the following code snippet, we use the ``orderBy()`` method to combine a
+descending sort on ``letter`` with an ascending sort on ``_id``.
+
+.. code-block:: java
+
+   import static com.mongodb.client.model.Sorts.orderBy;
+   import static com.mongodb.client.model.Sorts.ascending;
+   import static com.mongodb.client.model.Sorts.descending;
+
+   // <MongoCollection setup code here>
+
+   Bson orderBySort = orderBy(descending("letter"), ascending("_id"));
+   collection.find().sort(orderBySort);
+
+The code snippet above returns the documents in the
+:ref:`sort_example collection <sorts-builders-sort-example>`
+in the following order: 
+
+.. code-block:: json
+
+   {"_id": 1, "letter": "c", "food": "coffee with milk"}
+   {"_id": 6, "letter": "c", "food": "maple donut"}
+   {"_id": 4, "letter": "b", "food": "coffee with sugar"}
+   {"_id": 2, "letter": "a", "food": "donuts and coffee"}
+   {"_id": 3, "letter": "a", "food": "maple syrup"}
+   {"_id": 5, "letter": "a", "food": "milk and cookies"}
+
+Text Score
+----------
+
+If you are using the :manual:`$text </reference/operator/query/text/>`
+query operator to perform text searches on your collections of data you
+can sort the results of those text searches by
+:manual:`text score </reference/operator/aggregation/meta/#exp._S_meta>`
+(how well your search matched each returned document). To sort your search
+results by the text score associated with your text search so the best match
+is returned first, use the ``Sorts.metaTextScore()`` static factory method. Note
+that the ``$text`` query operator performs a text search on the content indexed
+by the :manual:`text index </core/index-text/>` of your collection. You must
+create a text index on your collection before you can run a text search
+and use ``Sorts.metaTextScore()``.
```

Comment:
I think these two sentences might make more sense in a "note" or "warn" block. Also, it can help to have a signal word like "Therefore," for the second sentence to show it's a consequence of the prior sentence.

Reply:
Thanks for the note. I agree that the signaling word 'therefore' is useful, and have added it.

I am not sure I agree that this should go in a note or warning block however (to me it feels fairly central to a discussion of text search), but I don't feel particularly strongly either way. I will move this content into a warning block, and if we find time would like to discuss this later and talk through this piece of feedback.

Fix: https://github.com/mongodb/docs-java/pull/59/commits/5252db66718a632fd71662919346136e58768e4e